commitlintplugins.test.ts

```diff
@@ -348,11 +348,36 @@ test('proper-issue-refs3', () => {
     let commitMsgWithHashtagRefInBlock =
         "foo: this is only a title" + "\n\n" + "Bar baz:\n\n```\ntype Foo = string #123\n```";
     let properIssueRefs3 = runCommitLintOnMsg(commitMsgWithHashtagRefInBlock);
-    console.log('HERE =======>' + properIssueRefs3.stdout)
     expect(properIssueRefs3.status).toBe(0);
 });
 
 
+test('proper-revert-message1', () => {
+    let commitMsgWithoutProperRevertMessage = 
+        'Revert "add abbreviations.ts"\n\n' + 
+        'This reverts commit 0272f587c7eece147e8d1756116b0b43e11c34ac.';
+    let properRevertMessage1 = runCommitLintOnMsg(commitMsgWithoutProperRevertMessage);
+    expect(properRevertMessage1.status).not.toBe(0);
+});
+
+
+test('proper-revert-message2', () => {
+    let commitMsgWithProperRevertMessage = 
+        'Revert "add abbreviations.ts"\n\n' + 
+        'This reverts commit 0272f587c7eece147e8d1756116b0b43e11c34ac\n' +
+        'because/otherwise bla bla.'
+    let properRevertMessage2 = runCommitLintOnMsg(commitMsgWithProperRevertMessage);
+    expect(properRevertMessage2.status).toBe(0);
+});
+
+
+test('proper-revert-message3', () => {
+    let commitMsgWithProperRevertMessage = 'Revert "add abbreviations.ts"';
```

Comment:
I said that it needs to contain the whole message from the previous test, to expand on it; I said message, not title

Reply:
ah damn I'm reading test3 not test2, sorry, let me read test2